libs/providers/ofrep-web/src/lib/ofrep-web-provider.ts

```diff
@@ -260,37 +269,41 @@ export class OFREPWebProvider implements Provider {
   }
 
   /**
-   * Evaluate is a function retrieving the value from a flag in the cache.
+   * _resolve is a function retrieving the value from a flag in the cache.
    * @param flagKey - name of the flag to retrieve
    * @param type - type of the flag
+   * @param defaultValue - default value
    * @private
    */
-  private evaluate<T extends FlagValue>(flagKey: string, type: string): ResolutionDetails<T> {
+  private _resolve<T extends FlagValue>(flagKey: string, type: string, defaultValue: T): ResolutionDetails<T> {
```

Comment:
It may be worth modifying the return type to ensure we always include flag metadata. This is how we handle it in flagd core.

https://github.com/open-feature/js-sdk-contrib/blob/68585ae9d95b70a0f445d4617fba7d0064a42e37/libs/shared/flagd-core/src/lib/flagd-core.ts#L15